includes/subscriptions/assets/js/product-page.js

```diff
@@ -0,0 +1,21 @@
+jQuery( function ( $ ) {
+	'use strict';
+
+	var wc_payments_product = {
+		init: function () {
+			// Workaround to prevent duplicate WCPay subscriptions when submit button is spammed in Chrome.
+			$( '#publish' ).on( 'click', function ( e ) {
+				if (
+					$( 'select#product-type option:selected' )
+						.text()
```

Comment:
We might want to use `.val()` here instead, using `.text()` might not work with l10n.

e.g. https://github.com/Automattic/woocommerce-subscriptions-core/blob/1.3.0/assets/js/admin/admin.js#L20

Reply:
Done in 6b18449.